### PR TITLE
bug 1633984 updating router cookie parameter

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1633984

### CODE REVIEW

File: architecture/networking/routes.adoc [hunk 1]
```diff
@@ -602,7 +602,7 @@ For all the items outlined in this section, you can set annotations on the
 |===
 |Variable | Description | Environment Variable Used as Default
 |`*haproxy.router.openshift.io/balance*`| Sets the load-balancing algorithm. Available options are `source`, `roundrobin`, and `leastconn`. | `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
-|`*haproxy.router.openshift.io/disable_cookies*`| Disables the use of cookies to track related connections. If set to `true` or `TRUE`, the balance algorithm is used to choose which back-end serves connections for each incoming HTTP request. |
+|`*router.openshift.io/cookie_name*`| Disables the use of cookies to track related connections. If set to `true` or `TRUE`, the balance algorithm is used to choose which back-end serves connections for each incoming HTTP request. |
 |`*haproxy.router.openshift.io/cookie_name*`| Specifies an optional cookie to be used for
```
Comment (lihongan) [inline architecture/networking/routes.adoc]: We should change this line `haproxy.router.openshift.io/cookie_name` to `router.openshift.io/cookie_name` but not the above one. I think the description in the bug may be confusing.

File: architecture/networking/routes.adoc [hunk 2]
```diff
@@ -602,7 +602,7 @@ For all the items outlined in this section, you can set annotations on the
 |===
 |Variable | Description | Environment Variable Used as Default
 |`*haproxy.router.openshift.io/balance*`| Sets the load-balancing algorithm. Available options are `source`, `roundrobin`, and `leastconn`. | `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
-|`*haproxy.router.openshift.io/disable_cookies*`| Disables the use of cookies to track related connections. If set to `true` or `TRUE`, the balance algorithm is used to choose which back-end serves connections for each incoming HTTP request. |
```
Comment (lihongan) [inline architecture/networking/routes.adoc]: The old one of disabling cookies is correct, please revert it.

File: architecture/networking/routes.adoc [hunk 3]
```diff
@@ -603,7 +603,7 @@ For all the items outlined in this section, you can set annotations on the
 |Variable | Description | Environment Variable Used as Default
 |`*haproxy.router.openshift.io/balance*`| Sets the load-balancing algorithm. Available options are `source`, `roundrobin`, and `leastconn`. | `ROUTER_TCP_BALANCE_SCHEME` for passthrough routes. Otherwise, use `ROUTER_LOAD_BALANCE_ALGORITHM`.
 |`*haproxy.router.openshift.io/disable_cookies*`| Disables the use of cookies to track related connections. If set to `true` or `TRUE`, the balance algorithm is used to choose which back-end serves connections for each incoming HTTP request. |
-|`*haproxy.router.openshift.io/cookie_name*`| Specifies an optional cookie to be used for
+||`*router.openshift.io/cookie_name*`| Specifies an optional cookie to use for
```
Comment (lihongan) [inline architecture/networking/routes.adoc]: seems extra `|` at beginning?

Comment (lihongan) [reply architecture/networking/routes.adoc]: other looks good. thanks

Comment (kalexand-rh) [reply architecture/networking/routes.adoc]: Yes. Fixed. :)

**Pull request merged**